cloudify_cli/tests/commands/test_secrets.py

```diff
@@ -90,25 +90,56 @@ def test_secrets_create_mutually_exclusive_arguments(self):
         )
         self.assertIn('mutually exclusive with arguments:', outcome.output)
 
-    def test_secrets_export_password_length(self):
+    def test_secrets_export_invalid_password_length(self):
         self.invoke('cfy secrets export -p 1234567',
                     err_str_segment='ERROR: Password must contain at least 8 '
-                                    'characters',
+                                    'characters.',
                     exception=CloudifyValidationError)
 
-    def test_secrets_export_password_empty(self):
-        self.invoke('cfy secrets export -p ' '',
-                    err_str_segment='2',
-                    exception=SystemExit)
+    def test_secrets_export_valid_password(self):
+        self.client.secrets.export = MagicMock(return_value={})
+        self.invoke('cfy secrets export -p 12345678')
+        call_args = list(self.client.secrets.export.call_args)
```

Comment:
Do we have to use `list`? what type is `call_args`?

Reply:
call_args is of type {_Call} which inherits from tuple, and that means we don't have to use list. 
After debugging the code now, I saw that I can use call_args without the string casting and it would work the same. Should I delete the list casting although it appears this way in different tests (like in tests_deployments)?